diff.py

```diff
@@ -2222,7 +2218,8 @@ def diff_sameline(old_line: Line, new_line: Line) -> None:
         # Probably regalloc difference, or signed vs unsigned
 
         # Compare each field in order
-        newfields, oldfields = new.split(","), old.split(",")
+        new_parts, old_parts = new.split(None, 1), old.split(None, 1)
+        newfields, oldfields = new_parts[1].split(","), old_parts[1].split(",")
```

Comment:
can we add a helper function for "split line into mnemonic + arguments"?

Reply:
sure